source/Plx.js

```diff
@@ -561,10 +561,10 @@ function getNewState(scrollPosition, props, state, element) {
   let lastSegmentScrolledBy = null;
   const maxScroll = Math.max(
     document.body.scrollHeight,
-    document.body.offsetHeight,
-    document.documentElement.clientHeight,
+    // document.body.offsetHeight,
+    // document.documentElement.clientHeight,
     document.documentElement.scrollHeight,
-    document.documentElement.offsetHeight
+    // document.documentElement.offsetHeight
   ) - window.innerHeight;
```

Comment:
I think you are right, we can simplify this (again I did this for IE), this seems to work in all browsers including IE10.
```js
  const bodyHeight = document.documentElement.scrollHeight || document.body.scrollHeight
  const maxScroll = bodyHeight - window.innerHeight;
```

Reply:
> I think you are right, we can simplify this (again I did this for IE), this seems to work in all browsers including IE10.
> 
> ```js
>   const bodyHeight = document.documentElement.scrollHeight || document.body.scrollHeight
>   const maxScroll = bodyHeight - window.innerHeight;
> ```

Good job!